projects/fsastorefrontlib/src/cms-components/seller-dashboard/seller-dashboard-list/seller-dashboard-list.component.html

```diff
@@ -59,9 +59,14 @@
     </table>
     <!-- Pagination -->
     <div class="row">
-      <ng-container *ngIf="customers.pagination.totalPages > 1">
-        <cx-pagination [pagination]="customers.pagination"></cx-pagination>
-      </ng-container>
+      <div class="col-6">
+        <ng-container *ngIf="customers.pagination">
```

Comment:
This `ng-container` seems redundant since customers is already checked in the first if container?